rmf_site_format/src/asset_source.rs

```diff
@@ -30,6 +30,7 @@ pub enum AssetSource {
     Search(String),
     Bundled(String),
     Package(String),
+    OSMSlippyMap(i32, f32, f32),
```

Comment:
What do you think of making this a struct variant instead of a tuple variant, e.g.
```
OSMSlippyMap{ zoom: i32, latitude: f32, longitude: f32 }
```
Otherwise I think it's unclear what each of the fields in the tuple is for.

Reply:
[fc2c182](https://github.com/open-rmf/rmf_site/pull/133/commits/fc2c182d5679b8cf20298741363d855b0c0b2e5d)